src/main/java/com/futurecollars/Task5/Pitagoras.java

```diff
@@ -0,0 +1,18 @@
+package com.futurecollars.Task5;
+
+public class Pitagoras {
+
+    public static String pitagoras(int a, int b, int c) {
+
+        if (a + b < c) {
+            return "Triangle does not exist";
+        } else {
+            if (a * a + b * b == c * c) {
+                return "Triangle is rectangular";
+            } else {
+                return "Triangle is not rectangular";
```

Comment:
Skoro mamy dwie opcje jakie ta metoda może zwrócić: trójkąt jest prostokątny/trójkąt nie jest prostokątny - to czy nie lepiej tutaj użyć inny typ danych zamiast Stringa? ;)

Reply:
wyobraź sobie, że teraz ktoś chce użyć tej metody, strasznie skomplikowane się to robi:

`if(pitagoras([...]).equals("Triangle is not rectangualr"))[...])`